src/model/cellStateHandlers/activatedBCell.gd

```diff
@@ -2,6 +2,7 @@ class_name ActivatedBCell extends CellStateHandler
 
 const DIFFERENCIATION_TRIGGER = Cell.TYPES.ANTIGENPRESENTINGCELL
 const DIFFERENCIATION_TARGET = Cell.TYPES.PLASMACYTE
+const MOVEMENT_TARGETS = [Cell.TYPES.ANTIGENPRESENTINGCELL]
```

Comment:
Why does the activated BCell move to an APC?

Reply:
It shouldn't, I will fix this.